app/helpers/application_helper.rb

```diff
@@ -1,2 +1,14 @@
 module ApplicationHelper
+  def rating_options(selected=nil)
+    options_for_select([["1 star", 1.0],["2 stars", 2.0],["3 stars", 3.0],["4 stars", 4.0],["5 stars", 5.0]], selected)
```

Comment:
You should find more dynamic way how to create the array of options. Programmer should not hardcode this stuff. Watch solutions.